src/lib.rs

```diff
@@ -86,12 +91,21 @@ pub fn test(attr: TokenStream, item: TokenStream) -> TokenStream {
     block,
   } = input;
 
-  let logging_init = expand_logging_init();
+  let mut non_test_log_attrs = vec![];
+  for attr in attrs {
+    let matched = populate_attribute_args(&mut attribute_args, &attr)?;
+    // Keep only attrs that didn't match the #[test_log(_)] syntax.
+    if !matched {
+      non_test_log_attrs.push(attr);
+    }
+  }
```

Comment:
Nit: Probably just a personal thing, but I find the logic somewhat hard to follow or at least review. Can we move the `attribute_args` definition closer to where it is used? And then would it make sense to just call a top level function here that parses all args and spits out the `AttributeArgs` object and the rest? And then perhaps rename `populate_attribute_args` to something containing `_single` or so to make it clearer that it's handling a single attribute?

Reply:
I moved it around and changed the function to be `attribute_args.try_parse_attribute_single(attr)`, lemme know wdyt